src/views/SWF.vue

```diff
@@ -206,16 +220,26 @@ export default {
     getTestData() {
       this.finalWeatherData = this.processWeatherData(testData, this.withTheseProps)
     },
-    getUserLoc() {
-      if (navigator.geolocation) {
-        navigator.geolocation.getCurrentPosition((position) => {
-          this.user_lat = position.coords.latitude
-          this.user_lng = position.coords.longitude
-        })
-      }
+    async getCoordinates() {
+      return new Promise(function(resolve, reject) {
+        navigator.geolocation.getCurrentPosition(resolve, reject);
+      });
     },
+    async useUserLoc() {
+      const autoCoords = await this.getCoordinates()
+      const gridCurLoc = await currentLocToGrid(
+        this.user_lat= autoCoords.coords.latitude,
```

Comment:
I didn't even know this could be done.  not sure how i feel about it.  I guess it's fine though.  I've never seen this declarations being done in params.

Reply:
all this is done, see if you get an alert